serviceconnector/src/createLinker/TargetServiceListStep.ts

```diff
@@ -43,7 +47,19 @@ export class TargetServiceListStep extends AzureWizardPromptStep<ICreateLinkerCo
             case TargetServiceTypeName.Storage:
                 promptSteps.push(new StorageAccountListStep(storageAccountCreateOptions));
                 break;
-            case TargetServiceTypeName.CosmosDB:
+            case TargetServiceTypeName.Cassandra:
```

Comment:
It might make more sense for the switch case to check the group. That shouldn't change anything for Storage (since all of the groups are also `TargetServiceTypeName.Storage`).

But it'd make this a lot cleaner since all are part of `TargetServiceTypeName.CosmosDB` and are all using the same promptStep as far as I can tell.

Reply:
Very true I will do that!